examples/AspNetCoreExample/Controllers/ValuesController.cs

```diff
@@ -28,7 +28,14 @@ public async Task<ActionResult<IEnumerable<string>>> Get()
 
             var val = this.r.Next();
             CompileMe(() => val);
-            
+
+            try
+            {
+                var divide = 0;
+                var result = 1 / divide;
```

Comment:
I appreciate the inclusion of this manual test but could you also include an automated test? See https://github.com/djluck/prometheus-net.DotNetRuntime/blob/master/src/prometheus-net.DotNetRuntime.Tests/StatsCollectors/IntegrationTests/ContentionStatsCollectorTests.cs as an example. The key things the test must do is:
1. Inherit from `StatsCollectorIntegrationTestBase<T>` where `T` would be the collector you're testing (i.e. `ExceptionStatsCollector `)
2. Add a test fixture that triggers a .NET event you are measuring and then verify the relevant counters on your stats collector were incremented with the correct labels